tests/Feature/Api/Assets/AssetIndexTest.php

```diff
@@ -30,7 +30,113 @@ public function testAssetIndexReturnsExpectedAssets()
             ->assertJson(fn(AssertableJson $json) => $json->has('rows', 3)->etc());
     }
 
-    public function testAssetIndexAdheresToCompanyScoping()
+    public function testAssetApiIndexReturnsDisplayUpcomingAuditsDue()
+    {
+        Asset::factory()->count(3)->create(['next_audit_date' => Carbon::now()->format('Y-m-d')]);
+
+        $this->assertTrue(Asset::count() === 3);
```

Comment:
Another quick note: I think we can remove these assertions and trust the factories worked as expected. If they stop working for some reason then most of the tests will blow up 😅